sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -179,6 +188,20 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
 
         .. versionadded:: 0.22
 
+    individual : "both" or bool, default=False
+        Whether to plot each individual partial dependence plots, the averaged
+        partial dependence plots or both.
```

Comment:
```suggestion
    kind : "average", "individual" or "both", default="average"
        Whether to plot the partial dependence averaged across all the samples
        in the dataset or one line per sample or both individual lines and the 
        average dependence at the same time.
        
        - kind="average" results in the traditional PD plot;
        - kind="individual" results in the ICE plot.

       Note that the fast method="recursion" option is only available for
       kind="average". Plotting individual dependencies requires using the slower
       method="brute" option.
```

Reply:
I am fine with parameter `kind="average"/"individual"/"both", default="average"`. I also proposed  something in that line in https://github.com/scikit-learn/scikit-learn/pull/16164#issuecomment-589512795.
@glemaitre @NicolasHug Do we have consensus?

Reply:
This is good for me.

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/f9fe5bc6ea565922bf369cc804771ed37785ae5a